.github/workflows/dockerimage.yml

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+name: Docker Image CI
```

Comment:
Maybe it's better to name it as `Next Dockerimage`?
Next here would mean - the latest changes from the master branch

Reply:
Ok. I have fixed it.